master/buildbot/test/unit/test_worker_openstack.py

```diff
@@ -40,6 +42,8 @@ class TestOpenStackWorker(unittest.TestCase):
     def setUp(self):
         self.patch(openstack, "nce", novaclient)
         self.patch(openstack, "client", novaclient)
+        self.build = Properties(
```

Comment:
I must have had more things planned/done to make it longer than 80 characters but then removed them. I thought it still was but just checked and it's well under. Oops, I should have checked earlier and I would have changed it. Oh well.